### DISCUSSION TITLE
Spanish translation opensis 7.4

### DISCUSSION THREAD

**Discussion by sfrivero** [OS4ED/openSIS-Classic#267] [General]
I have lang_sp.php file, my question is where do i have to upload the file to get the spanish to work in opensis 7.4

**Comment by rishtharamesh** [comment]
This is a feel good article if any of the business person are looking for assistance regarding there school management system. I found this site <a href=”https://www.ymtsindia.com/school-management-software ”>school management system</a> young minds technology solutions is the company name for more details visit their site.
